elevation_mapping/src/ElevationMapping.cpp

```diff
@@ -237,6 +238,20 @@ void ElevationMapping::visibilityCleanupThread()
 void ElevationMapping::pointCloudCallback(
     const sensor_msgs::PointCloud2& rawPointCloud)
 {
+  // Check if point cloud has corresponding robot pose at the beginning
+  if(not initialized_) {
```

Comment:
I did not know about the not keyword :D

Reply:
do you !like it?
I am asking since it is not resolved.

Reply:
Haha sorry, it is fine